src/shared/lib/OncoKbUtils.ts

```diff
@@ -1,60 +1,15 @@
+// TODO remove this file and completely rely on react-mutation-mapper implementation?
```

Comment:
Leaving this for another PR, since the portal cache implementation still needs these functions